auth/auth.js

```diff
@@ -0,0 +1,109 @@
+const router = require('express').Router();
+const passport = require('passport');
+import User from '../src/models/User/User';
+
+//Auth login
+router.get('/login', (req, res) => {
+  res.send(`  <form action="" method="post">
+  <div>
+      <label>Email</label>
+      <div>
+          <input type="text" name="email">
+      </div>
+  </div>
+  <div>
+      <label>Password</label>
+      <div >
+          <input type="password" name="password">
+      </div>
+  </div>
+  <input type="submit" value="Login">
+  <a href="/auth/signup" >Signup</a>
+</form>`);
+});
+//auth signup
+router.get('/signup', (req, res) => {
```

Comment:
this looks great.  I thought we weren't going to let users "sign-up?"  They were just going to log-in using their Spotify account?  I could be wrong ...